frontend/src/component/integrations/IntegrationForm/IntegrationMultiSelector/IntegrationMultiSelector.tsx

```diff
@@ -51,45 +42,30 @@ export const IntegrationMultiSelector: VFC<IIntegrationMultiSelectorProps> = ({
     error,
     onFocus,
     entityName,
-    selectAllEnabled = true,
     description,
+    note,
     required,
 }) => {
     const renderInput = (params: AutocompleteRenderInputParams) => (
         <TextField
             {...params}
             error={Boolean(error)}
-            helperText={error}
+            helperText={error || note}
```

Comment:
If this is error or note, maybe we should have a single text and another property of type `style: 'error'  || 'note'`? 
Personally, I don't like positional parameters having different meanings, and they seem to be mutually exclusive (i.e. when error is undefined you can have a note, when there's an error, it takes precedence over the note, so they can't be rendered together...